File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.py
class Solution(object):
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        count=0
        n=len(nums)
        for i in range(0,n):
            if nums[i]>nums[(i+1)%n]:
                count+=1
        if count<=1:
            return True
        else:
            return False
        

File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int dropCount=0;
        int n=nums.length;
        for(int i=0;i<n;i++){
            if(nums[i]>nums[(i+1)%n]){
                dropCount++;
            }
        }if(dropCount<=1){
            return true;
        }else{
            return false;
        }
        
    }
}